src/pages/task/TestsTableCore.tsx

```diff
@@ -0,0 +1,248 @@
+import React, { useEffect } from "react";
+import { ColumnProps, TableProps } from "antd/es/table";
+import { InfinityTable } from "antd-table-infinity";
+import { msToDuration } from "utils/string";
+import { Tag, Spin } from "antd";
+import { TESTS_QUERY } from "gql/queries";
+import { useParams, useLocation, useHistory } from "react-router-dom";
+import { useQuery } from "@apollo/react-hooks";
+import { usePrevious } from "hooks";
+import styled from "@emotion/styled/macro";
+
+import {
+  Categories,
+  RequiredQueryParams,
+  Sort,
+  ValidInitialQueryParams,
+  Limit,
+  TestStatus
+} from "pages/task/types";
+import get from "lodash.get";
+import queryString from "query-string";
+import { NetworkStatus } from "apollo-client";
+
+type Category = Categories.Duration | Categories.Status | Categories.TestName;
+
+interface TaskTestsData {
+  id: string;
+  status: string;
+  testFile: string;
+  duration: number;
+}
+
+type SortDir = "ASC" | "DESC";
+
+interface TakskTestsVars {
+  id: string;
+  dir: SortDir;
+  cat: Category;
+  pageNum: number;
+  limitNum: number;
+}
+
+interface UpdateQueryArg {
+  taskTests: [TaskTestsData];
+}
+
+const SpinWrapper = styled.div({
+  textAlign: "center",
+  paddingTop: 40,
+  paddingBottom: 40,
+  border: "1px solid #e8e8e8"
+});
+const loadMoreContent = (
+  <SpinWrapper>
+    <Spin tip="Loading..." />
+  </SpinWrapper>
+);
+
+const columns: Array<ColumnProps<TaskTestsData>> = [
+  {
+    title: "Name",
+    dataIndex: "testFile",
+    key: Categories.TestName,
+    sorter: true
+  },
+  {
+    title: "Status",
+    dataIndex: "status",
+    key: Categories.Status,
+    sorter: true,
+    render: (tag: string) => {
+      let color: string;
+      switch (tag) {
+        case TestStatus.Succeeded:
+          color = "green";
+          break;
+        case TestStatus.Failed:
+          color = "red";
+          break;
+        case TestStatus.SilentlyFailed:
+          color = "volcano";
+          break;
+        case TestStatus.Skipped:
```

Comment:
Let's use a cooler color for this - I'm thinking a shade of blue

Reply:
we should use whatever color is in designs